packages/adapter-vercel/index.d.ts

```diff
@@ -7,9 +7,8 @@ export interface ServerlessConfig {
 	/**
 	 * Whether to use [Edge Functions](https://vercel.com/docs/concepts/functions/edge-functions) (`'edge'`) or [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions) (`'nodejs18.x'`, `'nodejs20.x'` etc).
 	 * @default Same as the build environment
-	 * @deprecated
 	 */
-	runtime?: `nodejs${number}.x`;
+	runtime?: `nodejs${number}.x` | `experimental_bun1.x`;
```

Comment:
maybe we should change this while we're at it, IIUC we only support 20 and 22. this will give better autocomplete as well as being more accurate

```suggestion
	runtime?: `nodejs20.x` | `nodejs22.x` | `experimental_bun1.x`;
```